### PR TITLE
Fix updateat field

### PR DESCRIPTION
**Pull request by rande**

### CODE REVIEW

File: explorer/webpack.config.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ var config = {
             loader: "json"
         }]
     },
-    plugins: [
+    modules: [
```
Comment (plouc) [inline explorer/webpack.config.js]: I think you used an aggressive search and replace :-)

**Comment by plouc** [comment]
@rande, is this PR supposed to fix the updated_at auto-update or is it a refactoring of plugins ?

**Comment by rande** [comment]
frontend tests did not fail ... this is another story ;)

**Comment by plouc** [comment]
you're right, we need tests to prevent blind coding ;-P

**Comment by rande** [comment]
Well, the webpack command is not started on travis.

**Pull request merged**